lib/Models/prettifyCoordinates.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+import defined from 'terriajs-cesium/Source/Core/defined';
+import proj4 from 'proj4/lib/index.js';
```

Comment:
This file (and prettyifyProjection) should probably go in Maps instead.  It doesn't depend on `Terria`, knockout, or any of the other model-y stuff.